carousel/core/calculations.py

```diff
@@ -176,31 +83,41 @@ class Calc(object):
     __metaclass__ = CalcBase
 
     def __init__(self):
-        # XXX: Hack to get PR#68 done
-        if not self.parameters:
-            self.parameters = {
-                'always_calc': getattr(self, 'always_calc', False),
-                'frequency': getattr(self, 'frequency', [1, '']),
-                'dependencies': getattr(self, 'dependencies', []),
-                'static': getattr(self, 'static', []),
-                'dynamic': getattr(self, 'dynamic', [])
-            }
+        meta = getattr(self, CalcBase._meta_attr)
+        parameters = getattr(self, CalcBase._param_attr)
         #: ``True`` if always calculated (day and night)
-        self.always_calc = self.parameters.get('always_calc', False)
-        freq = self.parameters.get('frequency', [1, ''])
+        self.always_calc = dict.fromkeys(
```

Comment:
so the value of meta.always_calc will be applied to all parameters without distinction?